alacritty/src/input.rs

```diff
@@ -890,6 +898,13 @@ impl<'a, T: EventListener, A: ActionContext<T>> Processor<'a, T, A> {
             return;
         }
 
+        // Disable cursor blinking while the user is typing.
+        let cursor_blink_rate = self.ctx.config().cursor.blink_rate;
+        if let Some(timer) = self.ctx.scheduler_mut().get_mut(TimerId::BlinkCursor) {
+            timer.deadline = Instant::now() + Duration::from_millis(cursor_blink_rate);
+            self.ctx.show_cursor();
+        }
```

Comment:
Need something like that for Vi motions, since right now it's not really usable in it, since cursor constantly blinking while you're moving and you don't see the cursor.